### ISSUE TITLE
Is this repo still being maintained?

### ISSUE THREAD

**Issue by SamarRizvi** [buggedcom/phpvideotoolkit-v2#70]
I see many pull requests. Even I have submitted some. But there is no response from the author.

**Comment by buggedcom** [comment]
Yes but i haven't had the time to update it myself and have somehow missed
ghe pull requests. I will check them out after the holiday period

**Comment by SamarRizvi** [comment]
Thanks for letting know.

**Comment by SamarRizvi** [comment]
Hi @buggedcom ,
I have submitted some more pull requests, one of which is a major bugfix and other is an enhancement.

**Comment by tsmgeek** [comment]
I think this project has now been abandoned and may need forking.
Its great and very little needs to be done to since last release but really there are a few issues that need a new release.

#114
#112 #118 

And many other tickets need closing as they are so old and only requests for info.
